apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -186,7 +187,11 @@ const AssignContentTypeSection = (props: Props) => {
           )}
         </>
       ) : (
-        <Spinner variant="primary" />
+        <Flex marginTop="none" fullWidth={true}>
```

Comment:
is the marginTop of a flex container _not_ `none` by default? 🤔

Reply:
Good catch - copied this over from where the exact same component was implemented for the Property section since I wanted it to be consistent. Should have noticed that 👍